packages/flutter_tools/test/general.shard/build_system/targets/ios_test.dart

```diff
@@ -247,7 +253,7 @@ void main() {
     );
 
     await const ReleaseIosApplicationBundle().build(environment);
-    expect(processManager.hasRemainingExpectations, isFalse);
+    expect(processManager, hasNoRemainingExpectations);
```

Comment:
use `hasNoRemainingExpectations` instead of `processManager.hasRemainingExpectations, isFalse` to get helpful debugging logging about which expectations are remaining.

Reply:
Follow up: https://github.com/flutter/flutter/pull/108649